components/net/filemanager_thread.rs

```diff
@@ -59,50 +58,50 @@ impl FileManager {
             match self.receiver.recv().unwrap() {
                 FileManagerThreadMsg::SelectFile(sender) => self.select_file(sender),
                 FileManagerThreadMsg::SelectFiles(sender) => self.select_files(sender),
-                FileManagerThreadMsg::ReadFile(sender, id) => self.read_file(sender, id),
+                FileManagerThreadMsg::ReadFile(sender, id) => {
+                    match self.try_read_file(id) {
+                        Ok(buffer) => sender.send(Ok(buffer)).expect("FileManager send error"),
```

Comment:
Not `expect`, we should silently ignore the sending error.

`send()` fails if the receiver doesn't exist. This can happen if a content thread crashed, in this case. Content threads crashing should not bring down the chrome process.

(Basically, the chrome process should never crash unless it's an irrecoverable error for the entire browser)